code/game/gamemodes/cult/ritual.dm

```diff
@@ -370,7 +370,6 @@ var/global/list/rune_list = list() // HOLY FUCK WHY ARE WE LOOPING THROUGH THE W
 	else
 		M.LAssailant = user
 
-		return
```

Comment:
Yeah, but c'mon it's not a feature, just a bugfix, and a one-line change at that, and Pomf said that it was ok to include bugfixes with features as long as the features work fine.